src/main/java/gregtech/loaders/MaterialInfoLoader.java

```diff
@@ -13,281 +14,369 @@
+        OreDictUnifier.registerOre(new ItemStack(Blocks.QUARTZ_STAIRS, 1), new ItemMaterialInfo(new MaterialStack(Materials.NetherQuartz, (OrePrefix.block.getMaterialAmount(NetherQuartz) * 6) / 4)));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.BRICK_STAIRS, 1), new ItemMaterialInfo(new MaterialStack(Materials.Brick, (OrePrefix.block.getMaterialAmount(Brick) * 6) / 4)));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.NETHER_BRICK_STAIRS, 1), new ItemMaterialInfo(new MaterialStack(Materials.Netherrack, (OrePrefix.block.getMaterialAmount(Netherrack) * 6) / 4)));
+
+        // Meta 2 for stone slab just doesn't exist for some reason?
+        OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_SLAB, 1, 0), new ItemMaterialInfo(new MaterialStack(Materials.Stone, OrePrefix.block.getMaterialAmount(Stone) / 2)));
+        // TODO, Sandstone recycling
+        //OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_SLAB, 1, 1), new ItemMaterialInfo(new MaterialStack(Materials.Stone, M / 2)));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_SLAB, 1, 3), new ItemMaterialInfo(new MaterialStack(Materials.Stone, OrePrefix.block.getMaterialAmount(Stone) / 2)));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_SLAB, 1, 4), new ItemMaterialInfo(new MaterialStack(Materials.Brick, OrePrefix.block.getMaterialAmount(Brick) / 2)));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_SLAB, 1, 5), new ItemMaterialInfo(new MaterialStack(Materials.Stone, OrePrefix.block.getMaterialAmount(Stone) / 2)));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_SLAB, 1, 6), new ItemMaterialInfo(new MaterialStack(Materials.Netherrack, OrePrefix.block.getMaterialAmount(Netherrack) / 2)));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_SLAB, 1, 7), new ItemMaterialInfo(new MaterialStack(Materials.NetherQuartz, OrePrefix.block.getMaterialAmount(NetherQuartz) / 2)));
+
+        OreDictUnifier.registerOre(new ItemStack(Blocks.LEVER, 1, W), new ItemMaterialInfo(new MaterialStack(Materials.Stone, OrePrefix.block.getMaterialAmount(Stone)), new MaterialStack(Materials.Wood, OrePrefix.stick.getMaterialAmount(Wood))));
+
+        OreDictUnifier.registerOre(new ItemStack(Blocks.WOODEN_BUTTON, 1, W), new ItemMaterialInfo(new MaterialStack(Materials.Wood, OrePrefix.ingot.getMaterialAmount(Wood))));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.STONE_BUTTON, 1, W), new ItemMaterialInfo(new MaterialStack(Materials.Stone, OrePrefix.block.getMaterialAmount(Stone))));
+        OreDictUnifier.registerOre(new ItemStack(Blocks.REDSTONE_TORCH, 1, W), new ItemMaterialInfo(new MaterialStack(Materials.Wood, OrePrefix.stick.getMaterialAmount(Wood)), new MaterialStack(Materials.Redstone, OrePrefix.dust.getMaterialAmount(Redstone))));
+
+        // TODO, rails are a bit weird. Assembler doubles the output and has some different inputs
```

Comment:
May be best to remove the different input variants, and just standardize on the assembler output amount.